package.json

```
@@ -18,8 +18,8 @@
 		"@guardian/atom-renderer": "^2.0.0",
 		"@guardian/automat-modules": "^0.3.8",
 		"@guardian/braze-components": "^15.0.1",
-		"@guardian/commercial": "11.17.1",
-		"@guardian/consent-management-platform": "^13.6.1",
+		"@guardian/commercial": "11.17.2",
+		"@guardian/consent-management-platform": "^13.7.1",
```

[Our recommendations are to use exact, pinned versions](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript).

```suggestion
		"@guardian/consent-management-platform": "13.7.1",
```